03-base-r.Rmd

```diff
@@ -248,7 +248,7 @@ To resolve the usage issues described here, the tidymodels packages have a set o
 
 * Functions should take the **data structures that users have** as opposed to the data structure that developers want. For example, a model function's _only_ interface should not be constrained to matrices. Frequently, users will have non-numeric predictors such as factors. 
 
-Many of these ideas are described in the tidymodels guidelines for model implementation^[`https://tidymodels.github.io/model-implementation-principles`]. In subsequent chapters, we will illustrate examples of existing issues, along with their solutions. 
+Many of these ideas are described in the tidymodels guidelines for model implementation^[`[https://tidymodels.github.io/model-implementation-principles](https://tidymodels.github.io/model-implementation-principles)`]. In subsequent chapters, we will illustrate examples of existing issues, along with their solutions. 
```

Comment:
```suggestion
Many of these ideas are described in the tidymodels guidelines for model implementation^[https://tidymodels.github.io/model-implementation-principles]. In subsequent chapters, we will illustrate examples of existing issues, along with their solutions. 
```

I believe this will work. 👍

Reply:
You are Welcome @juliasilge !
Yes, after submitting the pull request I realised that all within backticks would probably fail, but I wanted to keep the style as in other links. For example, in 02-tidyverse.Rmd you have: [`https://design.tidyverse.org`](https://design.tidyverse.org/) that would also work and keep the teletype style.
cheers,
E.